packages/builder/src/components/backend/DataTable/TableDataTable.svelte

```diff
@@ -55,6 +59,7 @@
 <div class="wrapper">
   <Grid
     {API}
+    {isDark}
```

Comment:
Could we refactor this to be called `darkMode`? I think that's just a bit more standard than `isDark`.